packages/react-ui-components/src/SelectBox/selectBox.js

```diff
@@ -158,40 +156,50 @@ export default class SelectBox extends PureComponent {
 
     componentDidUpdate() {
         const listener = e => {
-            // Up and down arrow, enter
-            if ([38, 40, 13].indexOf(e.keyCode) > -1) {
+            const UP_ARROW = 38;
+            const DOWN_ARROW = 40;
+            const ENTER = 13;
+
+            // debouncing with lodash doesn't work as intended
+            // so I had to write my own debouncing function
+            const debounced = () => Date.now() - this.state.lastCall > 50;
```

Comment:
Denouncing via `setState`? But why? Isn't there a way to do it properly?

Reply:
https://github.com/neos/neos-ui/pull/1165/commits/cd67d221d7a8f1525f5a2c9912115445b956acf6#diff-46984f3d5a9788f9c39ad32b4dadc224R200 this doesn't work as I wanted it to :/